src/core/Form/Checkbox/Checkbox.baseStyles.tsx

```diff
@@ -0,0 +1,165 @@
+import { css } from 'styled-components';
+import {
+  withSuomifiTheme,
+  TokensAndTheme,
+  SuomifiThemeProp,
+} from '../../theme';
+import { disabledCursor } from '../../../components/utils/css';
+import { element, font } from '../../theme/reset';
+
+/* stylelint-disable no-descending-specificity */
+const checkedStyles = ({ theme }: SuomifiThemeProp) => css`
+  &.fi-checkbox--checked {
+    & .fi-checkbox_label {
+      &::before {
+        border-color: ${theme.colors.highlightBase};
+      }
+      & > .fi-checkbox_icon {
+        fill: ${theme.colors.highlightBase};
+      }
+    }
+  }
+`;
+
+const disabledStyles = ({ theme }: SuomifiThemeProp) => css`
+    &.fi-checkbox--disabled{    
+        & .fi-checkbox_label{
+            ${disabledCursor}
+            color: ${theme.colors.depthBase};
+        &::before{
+            background-color: ${theme.colors.depthLight3};
+            border-color: ${theme.colors.depthLight1};
+            border-width: 1px;
+        }
+        & > .fi-checkbox_icon {
+          fill: ${theme.colors.depthLight1};
+        }
+    }
+    &.fi-checkbox--large{
+      & .fi-checkbox_label::before{
+          border-width: 2px;
+        }
+      }
+    }
+`;
+
+const errorStyles = ({ theme }: SuomifiThemeProp) => css`
+  &.fi-checkbox--error {
+    & .fi-checkbox_label {
+      &::before {
+        border-color: ${theme.colors.alertBase};
+        border-width: 2px;
+      }
+      & > .fi-checkbox_icon {
+        fill: ${theme.colors.alertBase};
+      }
+    }
+    & .fi-checkbox_status {
+      color: ${theme.colors.alertBase};
+      margin-top: ${theme.spacing.xxs};
+    }
+  }
+`;
+
+const largeVariantStyles = ({ theme }: SuomifiThemeProp) => css`
+  &.fi-checkbox--large {
+    & .fi-checkbox_label {
+      padding-left: 40px;
```

Comment:
Would it be possible to use design token here? `spacing.xxl` is also 40px.

Reply:
Good find. This should now be fixed.